packages/core-ethereum/src/utils/index.ts

```diff
@@ -471,3 +471,11 @@ export function getSignatureParameters(
 export async function createChallenge(secretA: Uint8Array, secretB: Uint8Array): Promise<Hash> {
   return await hash(await hash(u8aConcat(secretA, secretB)))
 }
+
+/**
+ * @param network
+ * @returns true if network is private or ganache
+ */
+export function isGanache(network?: Network): boolean {
+  return !network || network === 'localhost'
```

Comment:
AFAIK we can set a network name in the Ganache config. This makes sure that we only run the Ganache-specific control-flow if we are really working with _our_ Ganache.